libr/core/core.c

```diff
@@ -1276,7 +1277,11 @@ R_API int r_core_init(RCore *core) {
 
 	r_core_config_init (core);
 
+	ut64 diff, prev = r_sys_now();
 	r_core_loadlibs_init (core);
+	diff = r_sys_now() - prev;
+	sdb_num_add(DB, "r_core_loadlibs_init time", diff, 0);
```

Comment:
@Maijin My bad, I noticed when I tried `?T` I get a valid output instead of `command not implemented` or smth similar so I assumed it was taken, I see now it ignores the T for the moment.
The file loading time question still stands btw.